compose/completion.md

```diff
@@ -78,7 +78,7 @@ Make sure you have [installed `oh-my-zsh`](https://ohmyz.sh/) on your computer.
 Add `docker` and `docker-compose` to the plugins list in `~/.zshrc` to run
 autocompletion within the oh-my-zsh shell. In the following example, `...`
 represent other Zsh plugins you may have installed. After that, type `source ~/.zshrc` to bring the changes. 
-To test whether it is successful, type `docker ps + tab`.
+To test whether it is successful, type `docker ps` + tab.
```

Comment:
Wondering if the "+ tab" is confusing; I think it's common to use something like;

```suggestion
To test whether it is successful, type `docker ps <TAB>`.
```

Reply:
To test whether it is successful, type `docker ps` and then press the Tab key.

Reply:
Yup, works for me (or `TAB` (formatted as code))